pkg/policy/rules/rules.go

```diff
@@ -0,0 +1,117 @@
+// Package rules contains useful pre-defined rego AST rules.
+package rules
+
+import "github.com/open-policy-agent/opa/ast"
+
```

Comment:
given that any of this functions may panic, and also is not really customizing any of the outputs, maybe a better approach would be to have those snippets in `var` section so that they'll all be evaluated on start. 

```go 
var (
    Session = ast.MustParseRule(`
get_session(id) = v {
	v := get_databroker_record("type.googleapis.com/user.ServiceAccount", id)
} else = v {
	v := get_databroker_record("type.googleapis.com/session.Session", id)
} else = v {
	v := {}
}
`)
)

Reply:
We modify the rules sometimes, so this would result in inadvertent data sharing.